<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtLangSetClass</class>
 <widget class="QMainWindow" name="QtLangSetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtLangSet</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RealCompare.qrc">
    <normaloff>:/Resources/edit/global/ndd.ico</normaloff>:/Resources/edit/global/ndd.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Language</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="langListWidget">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>User Define Language</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="userLangListWidget">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Style</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="styleListWidget">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Color</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Foreground:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_fc">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton">
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="modiryAllColor">
             <property name="text">
              <string>All Style</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>background:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_bc">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Same As Theme</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Font</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFontComboBox" name="fontComboBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QCheckBox" name="boldCheckBox">
               <property name="text">
                <string>Bold</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="italicCheckBox">
               <property name="text">
                <string>Italic</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="underlineCheckBox">
               <property name="text">
                <string>Underline</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Font Size:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="fontSpinBox">
                 <property name="singleStep">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="modifyAllFont">
               <property name="text">
                <string>All Style</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_normal">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Keyword And Mother </string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPlainTextEdit" name="keywordTe">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Mother Language:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="motherLangCb">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <item>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Cpp</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Ext File Type:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="extFileType">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_global">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Global Style Set</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QCheckBox" name="useGbFc">
               <property name="text">
                <string>Use Global Foreground Color</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="useGbBc">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Use Global Background Color</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="useGbFont">
               <property name="text">
                <string>Use Global Font</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="useGbFontSize">
               <property name="text">
                <string>Use Global FontSize</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="useGbFontBold">
               <property name="text">
                <string>Use Global Bold Font</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="useGbFontItalic">
               <property name="text">
                <string>Use Global Italic Font</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="useGbFontUnderline">
               <property name="text">
                <string>Use Global Underline Font</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="reset">
          <property name="text">
           <string>reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveBt">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelBt">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RealCompare.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelBt</sender>
   <signal>clicked()</signal>
   <receiver>QtLangSetClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>971</x>
     <y>694</y>
    </hint>
    <hint type="destinationlabel">
     <x>709</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBt</sender>
   <signal>clicked()</signal>
   <receiver>QtLangSetClass</receiver>
   <slot>slot_saveClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>857</x>
     <y>694</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>QtLangSetClass</receiver>
   <slot>slot_changeFgColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>832</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset</sender>
   <signal>clicked()</signal>
   <receiver>QtLangSetClass</receiver>
   <slot>slot_reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>743</x>
     <y>694</y>
    </hint>
    <hint type="destinationlabel">
     <x>814</x>
     <y>697</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useGbFc</sender>
   <signal>clicked(bool)</signal>
   <receiver>QtLangSetClass</receiver>
   <slot>slot_useAlobalFgColor(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useGbFont</sender>
   <signal>clicked(bool)</signal>
   <receiver>QtLangSetClass</receiver>
   <slot>slot_useAlobalFont(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useGbFontBold</sender>
   <signal>clicked(bool)</signal>
   <receiver>QtLangSetClass</receiver>
   <slot>slot_useAlobalFontBold(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useGbFontItalic</sender>
   <signal>clicked(bool)</signal>
   <receiver>QtLangSetClass</receiver>
   <slot>slot_useAlobalFontItalic(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useGbFontUnderline</sender>
   <signal>clicked(bool)</signal>
   <receiver>QtLangSetClass</receiver>
   <slot>slot_useAlobalFontUnderline(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>611</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useGbFontSize</sender>
   <signal>clicked(bool)</signal>
   <receiver>QtLangSetClass</receiver>
   <slot>slot_useAlobalFontSize(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useGbBc</sender>
   <signal>clicked(bool)</signal>
   <receiver>QtLangSetClass</receiver>
   <slot>slot_useAlobalBkColor(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_saveClick()</slot>
  <slot>slot_changeFgColor()</slot>
  <slot>slot_changeBkColor()</slot>
  <slot>slot_reset()</slot>
  <slot>slot_useAlobalFgColor(bool)</slot>
  <slot>slot_useAlobalFont(bool)</slot>
  <slot>slot_useAlobalFontSize(bool)</slot>
  <slot>slot_useAlobalFontBold(bool)</slot>
  <slot>slot_useAlobalFontUnderline(bool)</slot>
  <slot>slot_useAlobalFontItalic(bool)</slot>
  <slot>slot_useAlobalBkColor(bool)</slot>
 </slots>
</ui>
